src/ui/server/helpers/is-valid-req-original-url/index.ts

```diff
@@ -0,0 +1,26 @@
+import Joi from 'joi';
+import { REGEX } from '../../constants';
+
+/**
+ * Checks if the provided original URL is valid.
+ * checks against NUMBER_ALPHA_HYPHEN_DASH_QUESTION_EQUALS_AND regex
+ * if error on validation, returns false
+ * returns true if valid
+ * @param {string} originalUrl - The original URL to validate.
+ * @returns {Boolean}
+ */
+const isValidReqOriginalUrl = (originalUrl: string) => {
+  const joiString = Joi.string();
+
+  const schema = () => joiString.regex(REGEX.ALPHA_NUMBER_HYPHEN_DASH_QUESTION_EQUALS_AMPERSAND).required();
+
+  const validation = schema().validate(originalUrl);
+
+  if (validation.error) {
+    return false;
+  }
+
+  return true;
+};
+
+export default isValidReqOriginalUrl;
```

Comment:
Feels strange to have this new helper, and an existing `isValidWebsiteAddress` helper function that uses a `isUrl` method from `is-url`

Reply:
url will include https/http and .com etc whereas this is the bit after the url. Needs its own helper